Userland/Libraries/LibGfx/ImageFormats/ISOBMFF/JPEG2000Boxes.cpp

```diff
@@ -99,4 +99,23 @@ void JPEG2000SignatureBox::dump(String const& prepend) const
     outln("{}- signature = {:#08x}", prepend, signature);
 }
 
+ErrorOr<void> JPEG2000UUIDInfoBox::read_from_stream(BoxStream& stream)
+{
+    auto make_subbox = [](BoxType type, BoxStream&) -> ErrorOr<Optional<NonnullOwnPtr<Box>>> {
+        switch (type) {
+        default:
+            return OptionalNone {};
+        }
+    };
+
+    TRY(SuperBox::read_from_stream(stream, (ErrorOr<Optional<NonnullOwnPtr<Box>>>(*)(BoxType, BoxStream&))make_subbox));
```

Comment:
Isn't `ErrorOr<Optional<NonnullOwnPtr<Box>>>(*)(BoxType, BoxStream&)` the aliased callback type? Is that alias accessible from here?

Also, please no C-style casts.

Reply:
Turns out I just needed to `move()` the lambda. Thanks, this cast really bothered me.